bazel/external/quiche.BUILD

```diff
@@ -253,6 +256,48 @@ cc_library(
     deps = [":quic_platform_export"],
 )
 
+cc_library(
+    name = "epoll_server_platform",
+    testonly = 1,
+    hdrs = [
+        "quiche/epoll_server/platform/api/epoll_address_test_utils.h",
+        "quiche/epoll_server/platform/api/epoll_bug.h",
+        "quiche/epoll_server/platform/api/epoll_expect_bug.h",
+        "quiche/epoll_server/platform/api/epoll_export.h",
+        "quiche/epoll_server/platform/api/epoll_logging.h",
+        "quiche/epoll_server/platform/api/epoll_ptr_util.h",
+        "quiche/epoll_server/platform/api/epoll_test.h",
+        "quiche/epoll_server/platform/api/epoll_thread.h",
+        "quiche/epoll_server/platform/api/epoll_time.h",
+    ],
+    visibility = ["//visibility:public"],
+    deps = ["@envoy//test/extensions/quic_listeners/quiche/platform:epoll_server_platform_impl_lib"],
+)
+
+cc_library(
```

Comment:
Why are these `cc_library` rather than `envoy_cc_library`? The latter is preferred, it simplifies Google import and removes boilerplate.

Reply:
I think the original intention was that these targets are not in the envoy workspace and to use envoy_cc_library we need to explicitly specify `repository = "@envoy"`. But since it benefits to code import, I just switched to use envoy_cc_library.

Reply:
I guess internally we will use a different QUICHE binding actually. But, it's good either way.